### ISSUE TITLE
Adding components does not add to form in vs2017

### ISSUE THREAD

**Issue by danielj23** [ComponentFactory/Krypton#128]
I built the components and added them to the Toolbox, but every time I tried adding a component to a form, the component wouldn't appear on the form, but in the sub-area in Visual Studio.  (similar to adding a timer, it doesn't appear on the form itself)

I couldn't find a solution, so i removed all references to Krypton and then added the reference manually, and added ComponentFactory.Krypton.Design.dll

This allowed me to add components to the form properly.   I am not sure if this should behave this way but if anyone else is attempting and runs across it, this worked for me.

**Comment by Wagnerp** [comment]
Hi, I would recommend using [Krypton .NET 4.70](https://github.com/Wagnerp/Krypton-NET-4.70). As the issue was fixed in this repo. There are also installable pre-built binaries on there that you can use.

**Comment by danielj23** [comment]
Thanks.  I am only really using the toolkit, no other components.  any reason not to use the toolkit extended suite?

https://github.com/Wagnerp/Krypton-Toolkit-Suite-Extended-NET-4.70

**Comment by Wagnerp** [comment]
Not really, that repo is just an extension of the standard toolkit. Both I and @Smurf-IV felt that any extended tools/components wouldn't fit into the scope of the Krypton .NET 4.70 repo. All code in the extended toolkit might be unstable and subject to change.

**Comment by Wagnerp** [comment]
I've opened a pull request in which I backported this feature from the 4.70 version. We'll see what happens.
